perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AuthzResolverBlImpl.java

```diff
@@ -5,7 +5,22 @@
 import java.util.HashSet;
```

Comment:
Maybe I know why there is no interface for that class. Every method inside AuthzResolverBlImpl.java
is static method. So there is nothing to implement.